complex channel_0
complex channel_1
complex channel_2
complex noise
